docs/rag-setup.md

```diff
@@ -0,0 +1,32 @@
+# Setting up RAG using AI-services
+
+This section explains how to set up a RAG using the existing templates provided in AI-services.
+
+## Pull and Run the AI-services binary 
+
+To get started, follow the [installation guide](./installation.md) to pull and run the AI-services binary.
+
+## Create an App Using the RAG Template
+
+Initialize a new app using the built-in RAG template. It generates all essential resources needed to configure and run a RAG workflow.
+
+```bash
+$ ai-services application create <app-name> -t RAG --skip-validation numa --params UI_PORT=3000
```

Comment:
```suggestion
$ ai-services application create <app-name> -t RAG --params UI_PORT=3000
```
At prod, we should not suggest to skip any validation.
Can you please explain why do we need to pass `--params UI_PORT=3000`